inbox/util/blockstore.py

```diff
@@ -11,25 +11,21 @@
 # TODO: store AWS credentials in a better way.
 STORE_MSG_ON_S3 = config.get("STORE_MESSAGES_ON_S3", None)
 
-if STORE_MSG_ON_S3:
-    from boto.s3.connection import S3Connection
-    from boto.s3.key import Key
-else:
-    from inbox.util.file import mkdirp
-
-    def _data_file_directory(h):
-        return os.path.join(
-            config.get_required("MSG_PARTS_DIRECTORY"),
-            h[0],
-            h[1],
-            h[2],
-            h[3],
-            h[4],
-            h[5],
-        )
 
-    def _data_file_path(h):
-        return os.path.join(_data_file_directory(h), h)
```

Comment:
Previously this was conditionally enabled but I wanted to test both paths without starting separate processes. I think there is close to no value in importing this conditionally.